package.json

```diff
@@ -163,15 +152,13 @@
     "@typescript-eslint/eslint-plugin": "^5.62.0",
     "@typescript-eslint/parser": "^5.62.0",
     "autoprefixer": "^9.8.6",
-    "babel-core": "7.0.0-bridge.0",
     "babel-eslint": "^10.1.0",
```

Comment:
Couldn't change this to `@babel/eslint-parser` because that package won't work with our version of ESLint